library/keystone_user.py

```diff
@@ -0,0 +1,431 @@
+#!/usr/bin/python
```

Comment:
Why are we importing/forking our own library module of `keystone_user` instead of using the [upstream provided](https://github.com/ansible/ansible-modules-core/blob/devel/cloud/openstack/keystone_user.py) module already included in Ansible?